config/locales/ca.yml

```diff
@@ -1,8 +1,4 @@
 ca:
-  activemodel:
```

Comment:
why this removal?

Reply:
because this translation is already in Decidim, see https://github.com/CodiTramuntana/decidim-mallorca-app/pull/20#discussion_r802387383

Reply:
wow, good sight!